src/tools/illink/test/Mono.Linker.Tests.Cases/Attributes/OnlyKeepUsed/MethodWithUnmanagedConstraint.cs

```diff
@@ -1,3 +1,5 @@
+using System.Reflection;
+
```

Comment:
nit: remove extra line
```suggestion
using System.Reflection;
```